### ISSUE TITLE
How to close tutorial programmatically

### ISSUE THREAD

**Issue by angeloborrelli** [mhnowak/flutter_tutorial#4]
Hi,
I'm using flutter_tutorial plugin and find it very useful and simple-to-use. 
Regarding the possibility of closing the tutorial with a "Got It !" or other push button I don't find any way to make it...as far as i can see I'm able to implement next / previous buttons but not a "close" one.
Actually I'm using a kind of workaround simply assigning a new GlobalKey<TutorialWidgetState>() variable to the "key" property then this piece of code:
```
final GlobalKey<TutorialWidgetState> _tutorialKey = new GlobalKey<TutorialWidgetState>();
Tutorial().show(
  context,
  key: _tutorialKey,
  onPressedBehavior: OnPressedBehavior.none,
  ...
  dialogBuilder: (bcontext, index, next, _) {
     return Center(
      child: ElevatedButton(
             onPressed: () => _tutorialKey.currentState?.reverse(),
             child: Text("GOT IT!")
         )
     );
)
```
Basically it works but it doesn't reset `_isVisible` variable (inside tutorial.dart) so first time ok but after tapping the button it takes 2 taps to open tutorial again.
Guess a public `close()` function/callback would be perfect or maybe I'm missing something and would appreciate any help.
Thanks!

**Comment by mhnowak** [comment]
Hi, you're right with the current state of the package it's not very straightforward to do. It'd probably be useful to add a method similar to next and previous in dialog builder. 

For now tho I think the cleaner way of achieving what you want to do is: 
```dart
    _tutorialKey.currentState?.widget.close();
```

Let me know if it works. I haven't actually tried it. Just thinking that it might work.
